routeoptimization/snippets/src/main/java/com/example/OptimizeTours.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ *
+ * Create features in bulk for an existing entity type. See
+ * https://cloud.google.com/vertex-ai/docs/featurestore/setup
+ * before running the code snippet
+ */
+
+package com.example;
+
+// [START routeoptimization_v1_generated_RouteOptimization_OptimizeTours_sync]
+import com.google.maps.routeoptimization.v1.OptimizeToursRequest;
+import com.google.maps.routeoptimization.v1.OptimizeToursResponse;
+import com.google.maps.routeoptimization.v1.RouteOptimizationClient;
+import com.google.maps.routeoptimization.v1.Shipment;
+import com.google.maps.routeoptimization.v1.Shipment.VisitRequest;
+import com.google.maps.routeoptimization.v1.ShipmentModel;
+import com.google.maps.routeoptimization.v1.Vehicle;
+import com.google.type.LatLng;
+
+public class OptimizeTours {
```

Comment:
All code samples have to be executable. Please, add a static `main` method that invokes `optimizeTours` and handles all responses. No sophisticated error handling is required. However, if there is an error that is specific to this code sample, you can handle it separately. Please, follow instructions in go/code-snippets-201 to fence with the region tag _only_ import statements that are related to the code sample and the method `optimizeTourse`.
In order to have code snippet to show only the minimum of required code in documentation we ask to exclude all auxiliary methods (including `main`) from being shown in the code snippet itself. Readers can open code in Github in one click in order to see the whole code.